storybook/package.json

```diff
@@ -7,7 +7,7 @@
     "author": "Groupe Sharegate inc.",
     "license": "Apache-2.0",
     "scripts": {
-        "start-sb": "start-storybook -c .",
+        "start-sb": "start-storybook -c .  -p 9009",
```

Comment:
espace de trop + random de faire ça dans cette pr là

Reply:
Sure I will remove it but do a separate PR. It was annoying as I could not have a bookmark / refresh a page after a new start as the port would always change